src/win/tty.c

```diff
@@ -263,6 +289,7 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, uv_file fd, int unused) {
     tty->tty.rd.read_raw_wait = NULL;
 
     /* Init keycode-to-vt100 mapper state. */
+    tty->tty.rd.last_key = NULL;
```

Comment:
```suggestion
    tty->tty.rd.last_key = uv__malloc(32);
    if (tty->tty.rd.last_key == NULL)
      uv_fatal_error(ERROR_OUTOFMEMORY, "uv__malloc");
```

(and add free call to `uv_tty_endgame`)